### PR TITLE
Make sure .next cache is persisted in frontend CI

### PR DESCRIPTION
**Pull request by jeffin143**
Handling : https://github.com/opencollective/opencollective/issues/2642

### CODE REVIEW

**Comment by jeffin143** [comment]
> Maybe not the right place.
> 
> Look at replacing `node_modules/.cache/next-minifier`.

shoul i replace or append ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -125,7 +125,8 @@ jobs:
           paths:
             # In Next.js 8 it will be
             - ./node_modules/.cache/next-minifier
-
+            - ./node_modules
```
Comment (znarf) [inline .circleci/config.yml]: Why add `node_modules`? I don't think it's useful.

Should we remove `./node_modules/.cache/next-minifier`? Let's double check but I think it was actually replaced by `./.next/cache`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -125,6 +125,7 @@ jobs:
           paths:
             # In Next.js 8 it will be
             - ./node_modules/.cache/next-minifier
```
Comment (znarf) [inline .circleci/config.yml]: Don't think it's useful to keep `.cache/next-minifier`. No?

Comment (jeffin143) [reply .circleci/config.yml]: Yes, I will remove it and also squash it

**Comment by znarf** [comment]
It's not working :-/

Screenshot 2019-11-19 at 16 15 37

Screenshot 2019-11-19 at 16 15 46

**Comment by jeffin143** [comment]
> It's not working :-/ Not sure how to troubleshoot that.
> 
> Screenshot 2019-11-19 at 16 15 46
> 
> Screenshot 2019-11-19 at 16 15 37

We should go through it again I guess , I would take a look

**Comment by znarf** [comment]
@jeffin143 I nailed it. And now build are 2 times faster!

**Comment by jeffin143** [comment]
> @jeffin143 I nailed it. And now build are 2 times faster!

You are no less than a hero 🚀🚀

**Pull request merged**